package.json

```diff
@@ -102,6 +103,7 @@
   },
   "dependencies": {
     "@segment/analytics-node": "^1.1.0",
+    "chrome-remote-interface": "^0.33.0",
```

Comment:
This enables us to listen to CDP. Big thanks to [`cypress-har-generator`](https://github.com/NeuraLegion/cypress-har-generator) plugin, which uses this lib and was a good pattern to follow.